File: Practice Program/2DArray/2DArray/ViewController.m
//
//  ViewController.m
//  2DArray
//
//  Created by Anon on 9/12/14.
//  Copyright (c) 2014 Anthony Castor. All rights reserved.
//

#import "ViewController.h"

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad
{
    [super viewDidLoad];
     NSString *twoDimensionalArray[5][10];
	// Do any additional setup after loading the view, typically from a nib.
    twoDimensionalArray[0][0] = @"Hello1";
    twoDimensionalArray[0][1] = @"Hello2";
    twoDimensionalArray[0][2] = @"Hello3";
    twoDimensionalArray[0][3] = @"Hello4";
    twoDimensionalArray[0][4] = @"Hello5";
    
    
    NSLog(@"%@", twoDimensionalArray[0][0]);
    NSLog(@"%@", twoDimensionalArray[0][1]);
    NSLog(@"%@", twoDimensionalArray[0][2]);
    NSLog(@"%@", twoDimensionalArray[0][3]);
    NSLog(@"%@", twoDimensionalArray[0][4]);
    
    twoDimensionalArray[1][0] = @"1Hello1";
    twoDimensionalArray[1][1] = @"1Hello2";
    twoDimensionalArray[1][2] = @"1Hello3";
    twoDimensionalArray[1][3] = @"1Hello4";
    twoDimensionalArray[1][4] = @"1Hello5";
    
    NSLog(@"%@", twoDimensionalArray[1][0]);
    NSLog(@"%@", twoDimensionalArray[1][1]);
    NSLog(@"%@", twoDimensionalArray[1][2]);
    NSLog(@"%@", twoDimensionalArray[1][3]);
    NSLog(@"%@", twoDimensionalArray[1][4]);
}

- (void)didReceiveMemoryWarning
{
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated.
}

@end

File: NotificationsSwift/NotificationsSwift/ViewController.swift
//
//  ViewController.swift
//  NotificationsSwift
//
//  Created by Anon on 7/6/14.
//  Copyright (c) 2014 Anthony Castor. All rights reserved.
//

import UIKit

class ViewController: UIViewController {
                            
    override func viewDidLoad() {
        super.viewDidLoad()
        // Do any additional setup after loading the view, typically from a nib.
        NSNotificationCenter.defaultCenter().addObserver(self, selector:"drowAShape", name: "actionOnePressed", object: nil)
        NSNotificationCenter.defaultCenter().addObserver(self, selector:"showAMessage", name: "actionTwoPressed", object: nil)
        
        var dateComp:NSDateComponents = NSDateComponents()
        dateComp.year = 2014;
        dateComp.month = 07;
        dateComp.day  = 06
        dateComp.hour = 20;
        dateComp.minute = 41;
        dateComp.timeZone = NSTimeZone.systemTimeZone()
        
        var calendar:NSCalendar = NSCalendar(calendarIdentifier:NSGregorianCalendar)
        var date:NSDate = calendar.dateFromComponents(dateComp)
        
        var notification:UILocalNotification = UILocalNotification()
        notification.category = "FIRST_CATEGORY"
        notification.alertBody = "Hi, I am a notification"
        notification.fireDate = date
        
        UIApplication.sharedApplication().scheduleLocalNotification(notification)
    }
    
    func drowAShape(notification:NSNotification)
    {
        var view:UIView = UIView(frame:CGRectMake(10, 10, 100, 100))
        view.backgroundColor = UIColor.redColor()
        
        self.view.addSubview(view)
    }
    
    func showAMessage(notification:NSNotification)
    {
        var message:UIAlertController = UIAlertController(title: "A Notificaiton Message", message: "Hello there", preferredStyle: UIAlertControllerStyle.Alert)
        message.addAction(UIAlertAction(title: "OK", style: UIAlertActionStyle.Default, handler: nil))
        self.presentViewController(message, animated: true, completion: nil)
    }

    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
        // Dispose of any resources that can be recreated.
    }


}

